cdc/http_status_test.go

```diff
@@ -116,6 +118,20 @@ func testHandleChangefeedQuery(c *check.C) {
 	testRequestNonOwnerFailed(c, uri)
 }
 
+func testHandleChangefeeds(c *check.C) {
+	uri := fmt.Sprintf("http://%s/api/v1/changefeeds", advertiseAddr4Test)
```

Comment:
Could you also test ".../changefeed?state=normal|failed|stopped|removed|finished"

Reply:
I deleted the useless unit test and manually tested all the cases. 
I will use the new testing framework when I implement other HTTP APIs in the future.